db/BaseService/index.js

```diff
@@ -0,0 +1,57 @@
+const assert = require("assert");
+const { knex } = require("../helpers");
+
+class BaseService {
```

Comment:
I see the intention is to use this a baseClass that should be extended, but I think it is better to use composition and include this as a private variable. By using composition, you have more control over what is exposed and it also makes testing easier.  I wouldn't have the findXX{,s}By method exposed for example since I think it is better to have methods with clear input, so that you know which fields are actually queried and can make sure that query plan generated for is reasonable.